daily-coding-challenges/24-06-2020/repeatString.js

```diff
@@ -13,6 +13,9 @@
 
 const repeatStringNumTimes = (str, num) => {
 	/* Your implementation goes here */
+	if(num === 0) {
+		return null
```

Comment:
@iamfortune, this is good.. If `num` is zero, ideally you want to return the string not `null` right?